modules/lang-mustache/src/main/java/org/elasticsearch/script/mustache/CustomReflectionObjectHandler.java

```diff
@@ -39,6 +40,24 @@ public Object coerce(Object object) {
         }
     }
 
+    @Override
+    @SuppressWarnings("rawtypes")
```

Comment:
I think the `@SuppressWarnings` isn't needed here?

Reply:
Ehhhhh, gradle complains about the raw type `Class` in the args list there without it. And I can't change it to `Class<?>` because then it complains that I'm not overriding the signature of the superclass method.

If there's a different way through that works and satisfies gradle, I'm happy to drop the annotation.

Reply:
I commented this, because IntelliJ didn't complain about removing the annotation.
I didn't know gradle complains about this, so let's then just leave it there.